javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/instrumentation/internal/ExperimentalInstrumentationModule.java

```diff
@@ -25,4 +28,12 @@ public interface ExperimentalInstrumentationModule {
    * @param injector the builder for injecting classes
    */
   default void injectClasses(ClassInjector injector) {}
+
+  /**
+   * Returns a list of helper classes that will be defined in the class loader of the instrumented
+   * library.
+   */
+  default List<String> injectedClassNames() {
```

Comment:
WDYT about adding a method to the `ClassInjector` instead? We'd have a more consistent API for all class injection purposes this way

Reply:
Could this be addressed later? I think currently it is not clear yet whether this method should return a list a be more like `isHelperClass` that returns true/false based on the class name. Also it is unclear whether using a separate `ClassInjector` is needed or whether it would be enough to just provide it with a list of class names to proxy.

Reply:
Sure, that's fine by me 👍